bin/crew

```diff
@@ -1268,6 +1268,26 @@ def shrink_dir(dir)
   end
 end
 
+def install_files(src, dst = File.join( CREW_PREFIX, src.delete_prefix(".#{CREW_PREFIX}") ))
```

Comment:
It makes no sense to have `".#{CREW_PREFIX}"`.  That should be `"./usr/local"`, otherwise, why even delete the prefix?

Reply:
Changed.